core/objlist_unittest.cpp

```diff
@@ -168,5 +169,21 @@ TEST_F(TestObjList, WriteAndRead) {
         EXPECT_EQ(list_to_read.get_del(i), false);
 }
 
+TEST_F(TestObjList, EstimatedStorage) {
+    const size_t len = 1000 * 1000 * 10;
+    ObjList<Obj> test_list;
+    for (size_t i = 0; i < len; ++i) {
+        Obj obj(i);
+        test_list.add_object(std::move(obj));
+    }
+
+    const double sample_rate = 0.005;
+    size_t estimated_storage = test_list.estimated_storage_size(sample_rate);
+    size_t real_storage = test_list.get_data().capacity() * sizeof(Obj);  // just in this class Obj case
+    double diff_time = double(std::max(real_storage, estimated_storage))\
+        / double(std::min(real_storage, estimated_storage));
+    // EXPECT_LE(diff_time, 1);  // will be equal to 1 in this class Obj case
```

Comment:
You don't need to put a comment here. Otherwise, no sense for this unit test case.